### ISSUE TITLE
Voxbone Portal Static DIDs sync with Click2vox DB 

### ISSUE THREAD

**Issue by agonza1** [voxbone-workshop/click2vox-com#703]
Voxbone Portal Static DIDs - SIP URI assignation is not the same than in Click2vox DB.
Tasks:

[ ] - Clean Static DIDs: We have a lot of DIDs assigned to echo. And 6 or so assigned to digits. Do we need that?

[ ] - Once it is done we should make sure than the assigned DIDs in the voxbone portal are the same than in the DB. We might have to sync that data.

[ ] - Change of voxbonemarketing password. We don't know who is using the voxbonemarketing account and there are many assignations in the portal that haven't been done through click2vox.

[ ]- Using a new different account for staging and development. Voxbonemarketing is being used in staging and development too so some of the additional desynchronization might be due  to that. 

Bug #693 should be fixed after fixing this

**Comment by chadwallacehart** [comment]
I don't recommend doing a lot of work here. The whole mechanism of DID and SIP URI assignment is a hack. click2vox.com is a WebRTC to SIP service. We had to use Voxbone's DID/SIP URI mechanisms because that was the only option we had. Soon we'll have Janus and will be able to assign SIP addresses more directly without an associated DID. Migration to this system will certain necessitate a database schema upgrade and I recommend waiting until then.

`voxbonemarketing` is a shared account that was originally used for a few different applications. We can't unilaterally change the password. We could consider moving to a new account like `workshop_click2vox` that we can do whatever we want with, but I don't think that is worth the effort.

How DIDs have echo? I think we assigned `echo@ivrs` as the default SIP URI when we first started. We did not have a mechanism like did2webrtc/voxconf/voxrec where we determined available SIP URI's based on the assignment of a SIP URI to the DID. I guess that is what you want to do now?

**Comment by agonza1** [comment]
Yes, I spoke with @san983 and ideally we should change the methodology and be using API calls instead of using the local DB. However, we agree that this might be a big change that doesn't align with the plan of reducing click2vox cost and changing to Janus. 

So the minimal solution in order to be able to fix #693 and, at the same time, clean up some things for the Janus upgrade could be unassigning all those DID - `echo@ivrs` relations (except of the default click2vox static echo dids)

**Comment by san983** [comment]
I agree with both of you, @chadwallacehart @agonza1. Let's just clean up the voxbonemarketing account DIDs
